pkg/servicemesh/controller/extension/controller.go

```diff
@@ -68,14 +68,15 @@ func NewControllerFromConfigFile(kubeConfig string, namespaces []string, mrc mem
 				},
 			}
 		},
+		&v1alpha1.ServiceMeshExtension{}, resync,
 	)
 	if mrc != nil {
 		mrc.Register(mlw, "extensions-controller")
 	}
 	store := make(map[string]*v1alpha1.ServiceMeshExtension)
 	informer := cache.NewSharedIndexInformer(
 		mlw, &v1alpha1.ServiceMeshExtension{},
-		resync, cache.Indexers{})
+		0, cache.Indexers{})
```

Comment:
why does the resync period have to be `0` everywhere now? I see we're passing it into the `mlw` because it has its own informers now, just curios. does it break the `mlw`?

Reply:
The resync is handled by the lower level caches, there's no need to do it again in the higher cache